### ISSUE TITLE
`quai_nodeLocation` not supported

### ISSUE THREAD

**Issue by Juuddi** [PelagusWallet/pelagus-extension#123]
### Discord Discussion Link

https://discord.com/channels/1105993972247175229/1117868165771317459/1176210347074531348

### What browsers are you seeing the problem on?

Chrome

### What were you trying to do?

Attempting to set up injected provider using Pelagus with the `quais.js` package
1. Run development react application with `quais` installed and imported
2. Within application, set up quais Web3Provider with the following line
```
const provider = new quais.providers.Web3Provider(window.ethereum)
```
3. Check browser console for error

### What did not work?

When setting up the provider, quais.js requests 2 pieces of data from Pelagus: `chainId` and `nodeLocation`. Pelagus returns the `chainId` just fine, but does not support the method used to return the `nodeLocation` (`quai_nodeLocation`). This error is thrown in the browser and prevents the provider from functioning at all. 

### Version

None

### Relevant log output

```shell
content: inpage > background: {"id":"0","target":"tally-provider-bridge","request":{"method":"quai_accounts","params":[]}}
provider-bridge.js:1136  content: background > inpage: {"id":"0","jsonrpc":"2.0","result":["0xe674276466c99cd2589c11f475a990db9e29509d"]}
provider-bridge.js:1131  content: inpage > background: {"id":"1","target":"tally-provider-bridge","request":{"method":"quai_chainId","params":[]}}
provider-bridge.js:1131  content: inpage > background: {"id":"2","target":"tally-provider-bridge","request":{"method":"quai_nodeLocation","params":[]}}
provider-bridge.js:1131  content: inpage > background: {"id":"3","target":"tally-provider-bridge","request":{"method":"quai_chainId","params":[]}}
provider-bridge.js:1131  content: inpage > background: {"id":"4","target":"tally-provider-bridge","request":{"method":"quai_nodeLocation","params":[]}}
provider-bridge.js:1136  content: background > inpage: {"id":"2","jsonrpc":"2.0","result":{"code":4200,"message":"The Provider does not support the requested method."}}
index.js:224 Uncaught (in promise) Error: could not detect context (event="noNetwork", code=NETWORK_ERROR, version=providers/0.1.0)
    at Logger.makeError (index.js:224:1)
    at Logger.throwError (index.js:233:1)
    at Web3Provider.eval (json-rpc-provider.js:470:1)
    at Generator.throw (<anonymous>)
    at rejected (json-rpc-provider.js:6:42)
makeError @ index.js:224
throwError @ index.js:233
eval @ json-rpc-provider.js:470
rejected @ json-rpc-provider.js:6
Promise.then (async)
eval @ json-rpc-provider.js:385
setTimeout (async)
eval @ json-rpc-provider.js:384
JsonRpcProvider @ json-rpc-provider.js:383
Web3Provider @ web3-provider.js:123
useContract @ useContract.ts:12
Bet @ GameComponents.tsx:13
renderWithHooks @ react-dom.development.js:16305
mountIndeterminateComponent @ react-dom.development.js:20074
beginWork @ react-dom.development.js:21587
beginWork$1 @ react-dom.development.js:27426
performUnitOfWork @ react-dom.development.js:26557
workLoopSync @ react-dom.development.js:26466
renderRootSync @ react-dom.development.js:26434
performConcurrentWorkOnRoot @ react-dom.development.js:25738
workLoop @ scheduler.development.js:266
flushWork @ scheduler.development.js:239
performWorkUntilDeadline @ scheduler.development.js:533
provider-bridge.js:1136  content: background > inpage: {"id":"4","jsonrpc":"2.0","result":{"code":4200,"message":"The Provider does not support the requested method."}}
provider-bridge.js:1136  content: background > inpage: {"id":"1","jsonrpc":"2.0","result":"0x2328"}
index.js:224 Uncaught (in promise) Error: could not detect context (event="noNetwork", code=NETWORK_ERROR, version=providers/0.1.0)
    at Logger.makeError (index.js:224:1)
    at Logger.throwError (index.js:233:1)
    at Web3Provider.eval (json-rpc-provider.js:470:1)
    at Generator.throw (<anonymous>)
    at rejected (json-rpc-provider.js:6:42)
```

**Comment by Juuddi** [comment]
![Screenshot 2023-11-20 at 11 41 06 AM](https://github.com/PelagusWallet/pelagus-extension/assets/98721462/1cec5e0c-89b6-4ba0-87c7-808838edb91f)

**Comment by Juuddi** [comment]
@rajosch pinging you here for updates

**Comment by Juuddi** [comment]
closed by #124 
Chrome store update completed
